client/components/Planning/PlanningItem.tsx

```diff
@@ -244,7 +244,7 @@ class PlanningItemComponent extends React.Component<IProps, IState> {
                             </span>
                         )}
                     </Row>
-                    <Row>
+                    <Row style={{overflow: 'visible'}}> {/** overflow is needed for coverage icons */}
```

Comment:
We already have a `classes` prop for this component, you could use `sd-overflow--visible` here instead

Reply:
ok, I'll update it.